portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Jonathan Ward's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <div id="nav-bar" class="nav-bar">
```

Comment:
Can you think of a way to set up your navigation so it's only maintained in one place? If you wanted to add a new page to your navigation, for instance, there are a lot of places you would need to update.

Reply:
Like a way for it to have a central spot and be shared throughout all the code?